doc/source/contents/conf.rst

```diff
@@ -8,26 +8,51 @@ issuer
 
 The issuer ID of the OP, a unique value in URI format.
 
-----
-seed
-----
 
-Used in dynamic client registration endpoint when creating a new client_secret.
-If unset it will be random.
+--------------
+session params
```

Comment:
mmhhh ... this wouldn't be there because it's in a sibling PR already approved, so, I leave it as it is